tokio/src/runtime/queue.rs

```diff
@@ -13,9 +13,6 @@ use std::sync::atomic::Ordering::{AcqRel, Acquire, Release};
 /// Producer handle. May only be used from a single thread.
 pub(super) struct Local<T: 'static> {
     inner: Arc<Inner<T>>,
-
-    /// LIFO slot. Cannot be stolen.
-    next: Option<task::Notified<T>>,
```

Comment:
The LIFO slot optimization needed to be remove from the `queue` abstraction and moved into the `worker` as the scheduler now cares more about that detail.